packages/expo-sqlite/android/src/main/java/expo/modules/sqlite/SQLitePackage.java

```diff
@@ -0,0 +1,16 @@
+package expo.modules.sqlite;
+
+import android.content.Context;
+
+import java.util.Collections;
+import java.util.List;
+
+import expo.core.BasePackage;
+import expo.core.ExportedModule;
+
+public class SQLitePackage extends BasePackage {
+  @Override
+  public List<ExportedModule> createExportedModules(Context context) {
+    return Collections.singletonList((ExportedModule) new SQLiteModule(context));
+  }
+}
```

Comment:
I'm not sure other `.java` files end without an empty line at the end. Could you please prettify this file with the Android Studio formatter?

I also set up this macro on my computer to make the IDE prettify files automatically — https://stackoverflow.com/a/5581992/1123156. It's sometimes a little bit irritating, but `git add -i` helps me with unnecessary/unwanted changes to the files. 😅